### PR TITLE
Update debug.twig to include a link to the git project URL for the mailcow version tag

### PR DESCRIPTION
**Pull request by CallMeLeon167**
This update adds a link to the Git project URL for the Mailcow version tag in the Debug Twig file. The link allows to quickly access the corresponding version of the project and gather more information.

### CODE REVIEW

File: data/web/templates/debug.twig [hunk 1]
```diff
@@ -77,7 +77,7 @@
                         <td>Version</td>
                         <td class="text-break">
                           <div class="fw-bolder">
-                            <p ><a href="#" id="mailcow_version">{{ mailcow_info.version_tag }}</a></p>
+                            <p ><a href="{{ mailcow_info.git_project_url }}/releases/tag/{{ mailcow_info.version_tag }}" id="mailcow_version" target="_blanked">{{ mailcow_info.version_tag }}</a></p>
```
Comment (Fighter456) [inline data/web/templates/debug.twig]:
```suggestion
                            <p><a href="{{ mailcow_info.git_project_url }}/releases/tag/{{ mailcow_info.version_tag }}" id="mailcow_version" target="_blank">{{ mailcow_info.version_tag }}</a></p>
```

**Review by Fighter456 — COMMENTED** [review]
```target="_blanked"``` seems to be wrong.

**Comment by Fighter456** [comment]
The space within the `<p>`-opening element is superfluous. Can you fix it too?

**Review by Fighter456 — COMMENTED** [review]
👍

(Approval not possible due to repo-restrictions.)

**Review by DerLinkman — APPROVED** [review]

**Pull request merged**